draft-ietf-oauth-device-flow.xml

```diff
@@ -99,13 +97,16 @@
     <keyword>Smart Objects</keyword>
 
     <abstract>
-
       <t>
-	The device flow is suitable for OAuth 2.0 clients executing on
-	devices that do not have an easy data-entry method (e.g., game
-	consoles, TVs, picture frames, and media hubs), but where the end-user has separate access
-	to a user-agent on another computer or device (e.g., desktop computer, a
-	laptop, a smart phone, or a tablet).
+        This OAuth 2.0 authorization flow for browserless and input constrained
+        devices, often referred to as the device flow, enables OAuth clients to
+        request user authorization from devices that have an internet
+        connection, but don't have an easy input method (such as a smart TV,
+        media console, picture frame, or printer), or lack a suitable browser
+        for a more traditional OAuth flow. This authorization flow instructs the
+        user to action the authorization request on a secondary device, such as
+        a smartphone. There is no requirement communication between the
```

Comment:
This sentence sounds a bit strange to me: "There is no requirement communication between the constrained device and the user's secondary device."

Should this rather be read as "There is no requirement for communication between the constrained device and the user's secondary device."?